modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartDioClientCodegen.java

```diff
@@ -125,20 +125,13 @@ public String getHelp() {
 
     @Override
     public String toDefaultValue(Schema schema) {
-        if (ModelUtils.isMapSchema(schema)) {
```

Comment:
Does it make sense to remove these 4 lines from `DartClient` too?

Reply:
No, in the dart generator you need the collection default values. But in dart-dio there are none, we use a builder pattern there.

Reply:
The models are mutable and you said you would expect to have an empty collection that you can just add to after instantiating the object. This is what the collection default values are used for.

Reply:
>This is what the collection default values are used for.

Yes, when I worked on that I kept it for backwards compatibility, I would do it differently now: keep compatibility with most popular generators where default value is null, unless field is required or defined as non-null. At some point we'll have to break it, but not rush.